mmv1/products/iap/Settings.yaml

```diff
@@ -0,0 +1,278 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Settings'
+description: |
+  gcloud iap settings - manage IAP settings
+base_url: '{{name}}:iapSettings'
+self_link: '{{name}}:iapSettings'
+create_verb: :PATCH
+update_verb: :PATCH
+update_url: '{{name}}:iapSettings'
+import_format: ['{{name}}/iapSettings']
+id_format: '{{name}}/iapSettings'
+# Skipping the sweeper because instances will be deleted during cluster sweeps
+skip_sweeper: true
+
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    diff_suppress_func: 'NameDiffSuppress'
+    description: |
+      The resource name of the IAP protected resource.
+    required: true
```

Comment:
`immutable: true` is needed for this field, as each protected resource's  name is different.

Reply:
Added ``immutable: true`` to the name field